### PR TITLE
Fix #148 single source of truth for the changelog

### LINKED ISSUE #148: CHANGELOG.md should be the source of truth
We should setup something so that `CHANGELOG.md` is used in the doc and as a source for github releases.

### PR DESCRIPTION
**Pull request by johnraz**
This fixes #148 by making `CHANGELOG.md` the only source of truth for the doc.
It is achieved by symlinking it in the `docs` folder.

I added a docker utility to work on the doc + documentation in the README on how to build the doc (doc for doc 🙄)

Also fixed an easy deprecation warning in mkdocs.

### CODE REVIEW

File: mkdocs.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+MOUNT_FOLDER=/app
+MKDOCS_DEV_ADDR=${MKDOCS_DEV_ADDR-"0.0.0.0"}
+MKDOCS_DEV_PORT=${MKDOCS_DEV_PORT-"8000"}
+
+docker run --rm -it \
+    -v $(pwd):$MOUNT_FOLDER \
+    -w $MOUNT_FOLDER \
+    -p $MKDOCS_DEV_PORT:$MKDOCS_DEV_PORT \
+    -e MKDOCS_DEV_ADDR="$MKDOCS_DEV_ADDR:$MKDOCS_DEV_PORT" \
+    squidfunk/mkdocs-material:3.2.0 $*
```
Comment (sphrak) [inline mkdocs.sh]: Just making sure, `$*` is intended yes?

**Comment by sphrak** [comment]
@johnraz it looks good Once this has been merged I can port it to the keep-a-change-log format discussed previously in #161 :+1:

File: README.md [hunk 2]
```diff
@@ -50,3 +50,30 @@ Django versions.
...
+# Work on the documentation
+
+Our documentation is generated by [Mkdocs](https://www.mkdocs.org).
+
+You can refer to their documentation on how to install it or use `mkdocs.sh` in
+this folder.
+
+It will run mkdocs in a [docker](https://www.docker.com/) container.
+
+Running the script without any params the `serve` command.
```
Comment (belugame) [inline README.md]: This sentence doesn't quite make sense to me grammatically :thinking:

File: README.md [hunk 3]
```diff
@@ -50,3 +50,30 @@ Django versions.
...
+The server is exposed on localhost on port 8000.
+If you want to configure port serve is exposed on you can use the
```
Comment (belugame) [inline README.md]: Here I'm also not sure.. should it be "if you want to configure **a** port **where the** serve**r** is exposed on?

**Comment by belugame** [comment]
awesome work! thank you very much! :)

File: mkdocs.sh (hunk 1 above)
Comment (johnraz) [reply mkdocs.sh]: Yes sure it is :-)

**Comment by johnraz** [comment]
I will look at the grammar issues, did that while speaking to my wife, I may have missed some stuff 😄

**Pull request merged**